packages/server-core/src/user/inventory-item/inventory-item.model.ts

```diff
@@ -76,6 +76,26 @@ export default (app: Application): any => {
       through: models.user_inventory,
       foreignKey: 'inventoryItemId'
     })
+    ;(inventoryItem as any).hasMany(models.user_trade, { foreignKey: 'fromInventoryItemId1', required: true })
```

Comment:
this is definitely not the right way to implement this. this should only have one association with user_trade